src/Juvix/Compiler/Core/Transformation/Optimize/LetFolding.hs

```diff
@@ -0,0 +1,20 @@
+module Juvix.Compiler.Core.Transformation.Optimize.LetFolding where
```

Comment:
add a short description of the transformation at the top of the module

Reply:
done